core/executor.py

```diff
@@ -462,6 +465,7 @@ def RunCommandSub(self, cs_part):
         # don't use p.AddStateChange(process.SetPgid(...))
 
         r, w = posix.pipe()
+        p.SetBackground()
```

Comment:
Ditto - why is a CommandSub in the background?

Reply:
This (and the above) actually aren't necessary. I was being overly cautious. These aren't tracked by the jobs list, so they shouldn't trigger the `Done` log